CONTRIBUTING.md

```diff
@@ -2,55 +2,69 @@
 
 ![Content logo](demisto_content_logo.png)
 
-Welcome to Demisto content repo!
+Welcome to Demisto content repo! Contributions are welcome and appreciated.
 
 ## How to contribute
-
-To get you started, refer to our [Getting Started article](https://github.com/demisto/content/blob/master/docs)
-
-After you finish developing, there are two steps left before you can create a pull request:
-
- - Follow code conventions [here](https://github.com/demisto/content/tree/master/docs/code_conventions)
- - Run linting and test checks as detailed [here](https://github.com/demisto/content/tree/master/docs/linting)
- - Document your changes in the relevant changelog file as detailed [here](https://github.com/demisto/content/tree/master/docs/release_notes).
- - Validate files are formatted according correctly, by running from the Content root directory: ```PYTHONPATH="`pwd`:${PYTHONPATH}" python2 Tests/scripts/validate_files.py```
- - Make sure you have test playbook [here](https://github.com/demisto/content/tree/master/docs/tests)
- - Make sure you have documentation [here](https://github.com/demisto/content/tree/master/docs/integration_documentation)
+Please read the following guidelines. Following these guidelines will maximize the chances for a fast, easy and effective review process for everyone involved. If something is not clear, please don't hesitate to reach out to us via GitHub, Slack, or email.
+
+* Setup a development environment by following the [Getting Started Guide](docs/getting_started).
+* Use the [Package Directory Structure](docs/package_directory_structure) for all Python code. If working on existing code, beyond trivial changes, we require converting to this structure as it allows running linting, unit tests and provides a clearer review process.
+* Make sure to read and follow [code conventions](docs/code_conventions).
+* Run and verify that the various linters we support pass as detailed [here](docs/linting).
+* For Scripts/Integrations make sure to create unit tests as documented [here](docs/tests/unit-testing)
+* Create a test playbook as documented [here](docs/tests). **Note**: for small Scripts that have unit tests, a test playbook is optional.
```

Comment:
not sure if `small Scripts` is the way to describe these, maybe "simple" or "straight-forward"